outputPortfolio.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * Turns the database list of portfolio items into html displaying the items
+ *
+ * @param array $data is the array containing all the portfolio items
+ *
+ * @return string is the html containing the portfolio items
+ */
+function outputPortfolio (array $data) : string {
+    $echoedString = '';
+    foreach ($data as $row) {
+        if (count($row) != 5) {
+            return 'Invalid array passed';
+        }
+        $echoedString .= '
```

Comment:
I would consider using a " in place of ' so you can use \n to generate line breaks in your $echoedString.